everyBot/cogs/fun/fun.py

```diff
@@ -54,18 +54,25 @@ async def eightball(self, ctx, *, question: commands.clean_content):
         ]
         # Choose random reply
         response = random.choice(responses)
-        await ctx.send(f':8ball:: "{ question }" { response }')
+        return await ctx.send(f':8ball:: "{ question }" { response }')

```

Comment:
I'd like to move towards all responses (excluding images) being discord embeds. They can be formatted much more and look much nicer

Reply:
@Givo29 Check changes. Is this the correct format for using embeds?